### PR TITLE
Improvements/table component

### PR DESCRIPTION
**Pull request by Raehan248**
- Completed the table component

### CODE REVIEW

File: src/components/Table/Table.module.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+@import "../../sharedStyles.scss";
+@import "../../colours.scss";
+
+.tableHeader {
+  display: flex;
+  margin-bottom: 1rem;
+}
+
+.controls {
+  display: flex;
```
Comment (tahaNusoft) [inline src/components/Table/Table.module.scss]: user this from bootstrap please and make sure that this approach is not being used anywhere please

File: src/components/Table/Table.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+const SimpleTable: React.FC<inputProps> = ({
+  columns,
+  data,
+  title,
+  sortByOptions,
+}) => {
+  const [filteredData, setFilteredData] = useState(data);
+
+  const handleSearch = (value: string) => {
```
Comment (tahaNusoft) [inline src/components/Table/Table.tsx:25]: what if i don't want want to sort initially ? will the data be shown ?

File: src/components/Table/Table.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+  const handleSort = (value: string) => {
+    // Implement sorting logic here
+  };
+
+  return (
+    <Content className="p-2">
+      <Row
+        className={`${styles.tableHeader}`}
+        style={{ alignItems: "center", justifyContent: "space-between" }}
```
Comment (tahaNusoft) [inline src/components/Table/Table.tsx]: remove inline styling and use it with bootstrap please

File: src/components/Table/Table.tsx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+      >
+        <Col className={styles.title}>
+          <Text>{title}</Text>
+        </Col>
+        <Col
+          className={`${styles.controls}`}
+          style={{
```
Comment (tahaNusoft) [inline src/components/Table/Table.tsx]: why inline ? any reason ?

File: src/components/Table/Table.tsx (hunk 2 above)
Comment (Raehan248) [reply src/components/Table/Table.tsx:25]: yes data is being shown initially before sorting as well in storybook file, further testing can be done in actual projects when we have more data.

**Review by tahaNusoft — APPROVED** [review]

**Pull request merged**